firmware/raspi/status.py

```diff
@@ -32,6 +32,12 @@ def __str__(self):
             return "ERROR"
         return ""
 
+    def is_valid_code(self, code):
+        return code in (ArduinoStatus.IDLE,
```

Comment:
Can you please update the spacing here so that all the `ArduinoStatus.*` start at the same alignment?